packages/core/src/components/forms/_file-upload.scss

```diff
@@ -16,12 +16,16 @@ Markup:
 </label>
 
 :disabled - Disabled
+.pt-large - Larger size
 .pt-fill - Take up full width of parent element
 
 Styleguide pt-file-upload
 */
 
+$file-upload-input-selector: ".pt-file-upload-input";
```

Comment:
you could do simply `.pt-large &` inside the original styles for `.pt-file-upload-input` to avoid repeating the selector. this is how buttons work--any large parent (not just `.pt-button-group`) will make nested buttons large.